### PR TITLE
Repoclosure for modules

### PR DESCRIPTION
**Pull request by j-mracek**
Requires: https://github.com/rpm-software-management/libdnf/pull/1005

### CODE REVIEW

File: plugins/repoclosure.py [hunk 1]
```diff
@@ -53,20 +54,239 @@ def configure(self):
...
+            problem = self._analyze_modular_combinations(
+                arch, combinations, base_query, module_dict, non_modular)
+            if problem or broken_module_dict:
+                self._raise_unresolved_dependencies()
         else:
-            unresolved = self._get_unresolved()
+            unresolved = self._get_unresolved(arch, self.base.sack.query().available(),
+                                              self.base.sack.query().available())
+            self._report_results_to_terminal(unresolved)
+            if len(unresolved) > 0:
```
Comment (sandrobonazzola) [inline plugins/repoclosure.py]: Why not just "if unresolved:" ?

**Comment by sandrobonazzola** [comment]
@j-mracek can you please comment on why this got closed?

**Comment by j-mracek** [comment]
I am sorry, I forget to add a description. The reason is that DNF is going to be replaced by DNF5 in Fedora 39. Therefore we will create another implementation for DNF5 anyway. Do you think it make sense to get the feature to current code base?

**Comment by sandrobonazzola** [comment]
No strong feelings, it would still be useful if backported to RHEL8 or RHEL9 but on Fedora I don't see a need.

**Comment by j-mracek** [comment]
@sandrobonazzola Ok, for RHEL we need a bug report on bugzilla.

**Comment by sandrobonazzola** [comment]
I see someone already did, is https://bugzilla.redhat.com/show_bug.cgi?id=2183091 enough?

**Comment by j-mracek** [comment]
> I see someone already did, is https://bugzilla.redhat.com/show_bug.cgi?id=2183091 enough?

This is something else. It is about presence of fail safe metadata on the system

**Pull request closed without merging**